cmake/finds/RequiredDependencySearch.cmake

```diff
@@ -68,7 +68,6 @@ GetDependency(OTF2)
 set(HAVE_HDF5 1)
```

Comment:
is HAVE_HDF5 needed?

Reply:
I was actually just checking this, same thing for NLopt, they don't appear to be used.

Reply:
please remove it

Reply:
Removed in https://github.com/NexGenAnalytics/MIT-MUQ/pull/32

Reply:
i mean remove in this PR the HAVE_HDF5